src/components/PokemonCard.js

```diff
@@ -0,0 +1,51 @@
+import React, { useState, useEffect } from 'react';
+import axios from "axios";
+import "../App.css"
+
+function PokemonCard( {url} ) {
+    const [pokemonCard, setPokemonCard] = useState({})
+    const [abilities, setAbilities] = useState([])
+    const [moves, setMoves] = useState("")
+    const [image, setImage] = useState("")
+
+    useEffect(() => {
+        async function fetchData() {
+            try {
+                const { data } = await axios.get(url)
+                setPokemonCard(data)
+                setMoves(data.moves)
+                setImage(data.sprites)
+                setAbilities(data.abilities)
```

Comment:
Deze zijn in principe overbodig, omdat je die nu ook van `pokemonCard` kan gebruiken: `pokemonCard.moves` bijv.